src/pages/signup/First.tsx

```
@@ -1,37 +1,103 @@
 import { Button } from '../../components/Button'
+import { MailPayload, SignupFormType } from '../../apis/user/type'
+import { confirmMailCode, requestMailVerification } from '../../apis/auth/mail'
+
+type SetStateType = { setStep: Dispatch<SetStateAction<number>>; setForm: Dispatch<SetStateAction<SignupFormType>> }
+
+function First({ setStep, setForm }: SetStateType) {
+  const [input, setInput] = useState<MailPayload>({ mail: '', code: '' })
+  const [errors, setErrors] = useState<MailPayload>({ mail: '', code: '' })
+  const [isSent, setIsSent] = useState<boolean>(false)
+
+  const mailRegExp = /^[a-zA-Z0-9+-\_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/
```

ESLint

[error] Unnecessary escape character: \_.

(no-useless-escape)